.github/workflows/build_windows_exp3.yml

```
@@ -0,0 +1,40 @@
+    paths:
+      - 'exp3/**'       # 监听代码修改
+      - '.github/workflows/build_windows_exp3.yml'
+
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: windows-latest
+
+    steps:
+    - uses: actions/checkout@v3
```

actionlint (1.7.8)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)